src/main.js

```diff
@@ -4,312 +4,174 @@ var bookTitle = document.querySelector('.cover-title');
-
-// Create your event handlers and other functions here 👇
-function getRandomIndex(array) {
-  return Math.floor(Math.random() * array.length);
-}
-
 var currentCover;
-function displayNewCover(placeHolder) {
-  bookCover.src = placeHolder.cover || covers[getRandomIndex(covers)];
-  bookTitle.innerText = placeHolder.title || titles[getRandomIndex(titles)];
-  firstDescriptor.innerText = placeHolder.tagline1 || descriptors[getRandomIndex(descriptors)];
-  secondDescriptor.innerText = placeHolder.tagline2 || descriptors[getRandomIndex(descriptors)];
-  currentCover = new Cover(bookCover.src, bookTitle.innerText, firstDescriptor.innerText, secondDescriptor.innerText);
-  }
-
-// if (2 > 1) {
-//   currentCover = new Cover(covers[0], titles[0], descriptors[0], descriptors[1]);
-// } else {
 
-//Iteration 2:
 
-//Form-View Functionality:
-var makeNewButton = document.querySelector('.make-new-button');
+// Add your event listeners here 👇
+window.addEventListener("load", displayNewCover);
+randomButton.addEventListener("click", displayNewCover);
 makeNewButton.addEventListener('click', toggleMakeOwn);
-
-
-function toggleMakeOwn() {
-  document.querySelector('.random-cover-button').classList.add('hidden');
-  document.querySelector('.save-cover-button').classList.add('hidden');
-  document.querySelector('.home-button').classList.remove('hidden');
-  document.querySelector('.form-view').classList.remove('hidden');
-  document.querySelector('.home-view').classList.add('hidden');
-  document.querySelector('.saved-view').classList.add('hidden');
-}
-
-
-
-//Save-Cover Functionality:
-
-var savedCoversButton = document.querySelector('.view-saved-button');
-savedCoversButton.addEventListener('click', clickViewSave);
-
-
-function toggleSavedCovers() {
-  document.querySelector('.saved-view').classList.remove('hidden');
-  document.querySelector('.random-cover-button').classList.add('hidden');
-  document.querySelector('.save-cover-button').classList.add('hidden');
-  document.querySelector('.home-button').classList.remove('hidden');
-  document.querySelector('.home-view').classList.add('hidden');
-  document.querySelector('.form-view').classList.add('hidden');
-}
-
-//Home Button Functionality
-
-var homeButton = document.querySelector('.home-button');
 homeButton.addEventListener('click', toggleHomeButton);
-
-
-function toggleHomeButton() {
-  document.querySelector('.saved-view').classList.add('hidden');
-  document.querySelector('.random-cover-button').classList.remove('hidden');
-  document.querySelector('.save-cover-button').classList.remove('hidden');
-  document.querySelector('.home-button').classList.add('hidden');
-  document.querySelector('.home-view').classList.remove('hidden');
-  document.querySelector('.form-view').classList.add('hidden');
-}
-
-
-// Iteration 3:
-var makeNewBookButton = document.querySelector('.create-new-book-button');
-
+savedCoversButton.addEventListener('click', clickViewSave);
```

Comment:
Should we nest our add and remove 'hidden' function within an anonymous function? Could cut down on the raw amount of functions.